### PR TITLE
Use default images in new_config.sh.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

File: scripts/new-config.sh [hunk 1]
```diff
@@ -28,9 +28,7 @@ cat > examples/local/${CONFIG_NAME}.json <<'EOF'
     "shared_filesystem": "",
     "use_gcs_key_secret": false,
     "use_tensorboard": false,
-    "user": "root",
-    "tf_image": "mesosphere/dcos-tensorflow:v1.3",
-    "gpu_tf_image": "mesosphere/dcos-tensorflow:v1.3-gpu"
+    "user": "root"
```
Comment (elezar) [inline scripts/new-config.sh:31]: For a follow-up we may want to consider looking into whether the default user `nobody` could also be used?

**Review by elezar — APPROVED** [review]
One unrelated comment.

File: scripts/new-config.sh (hunk 1 above)
Comment (joerg84) [reply scripts/new-config.sh:31]: @springle Do you know?

Comment (springle) [reply scripts/new-config.sh:31]: I don't know much about this field, but I will test the framework with `"user": "nobody"` and make a separate PR if there are no issues with the switch.

**Pull request merged**